packages/manager/src/features/Linodes/LinodeCreatev2/Actions.tsx

```diff
@@ -1,18 +1,24 @@
-import { CreateLinodeRequest } from '@linode/api-v4';
 import React, { useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 
 import { Box } from 'src/components/Box';
 import { Button } from 'src/components/Button/Button';
+import { useFlags } from 'src/hooks/useFlags';
 import { useRestrictedGlobalGrantCheck } from 'src/hooks/useRestrictedGlobalGrantCheck';
 import { scrollErrorIntoView } from 'src/utilities/scrollErrorIntoView';
 
 import { ApiAwarenessModal } from '../LinodesCreate/ApiAwarenessModal/ApiAwarenessModal';
 import { getLinodeCreatePayload } from './utilities';
 
+import type { CreateLinodeRequest } from '@linode/api-v4';
+
 export const Actions = () => {
+  const flags = useFlags();
+
   const [isAPIAwarenessModalOpen, setIsAPIAwarenessModalOpen] = useState(false);
 
+  const isFeatureEnabled = flags?.apicliDxToolsAdditions;
```

Comment:
Can we use a more descriptive name here?

maybe something like
```suggestion
  const isDxToolsAdditionsEnabled = flags?.apicliDxToolsAdditions;
```

Reply:
Looks like i missed this file in initial name change... update in b651370